### PR TITLE
Support request system

### PR DESCRIPTION
**Pull request by HitoriWoB**
This finishes #140 
Also included is misc. added support for static pages to display user preferred layout, will change if this is not wanted.

### CODE REVIEW

**Comment by HitoriWoB** [comment]
- Removed staff assignments
- Removed notifications, shouldn't be a problem with active staff
- Support types and priorities were made static
- Added quick reply to responses
- Corrected email domains
- Added support for red username display for user admins (permission to use was never removed, display was set to head admin only)

**Comment by HitoriWoB** [comment]
Since this incorporates more than jus the support system.
NEWS POST:

`
[b]DATE[/b]
[Hitori]
[b]Added Support System[/b]
Guest supports - Can be submitted without access to your account, check your email for staff updates on your support
*** Once account access is regained, you can add the guest support to your account by providing the support key on the support page to continue the support. This will change the support type to a user support

User supports - Submitted while logged into the game. Notifications for staff updates will be provided through PMs

Premium supports - Submitted while logged into the game, costs anywhere between 0 and 10 AK to submit. Submitting a premium support provides higher priority to your request. 
[i]Functionality Note:[/i] A non-priority premium support (e.g. Game Question) will still have a lower priority than a high-priority user support (e.g. Account Problem). However, your support will still be processed at a premium priority after high-priority issues are processed.

[b]Misc. Changes[/b]
Added support to static pages (e.g. rules, manual, etc.) to support pages to use user selected layouts while logged in.
Added more feedback for account activation errors. You will now be notified if your account is already verified instead of directing to administrators.
`

File: classes/SupportManager.php [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+                'Report Bug',
+                'Game Question',
+                'Content Text Issue',
+                'Balance Feedback',
+                'Report Staff Member',
+                'Appeal Ban',
+                'Misc. Request',
+                'Suggestion',
+            ];
+            if($staffLevel >= System::SC_MODERATOR) {
```
Comment (levimeahan) [inline classes/SupportManager.php:126]:
```suggestion
            if($staffLevel == System::SC_MODERATOR) {
```

File: classes/SupportManager.php [hunk 2]
```diff
@@ -0,0 +1,689 @@
...
+    public static $strfString = '%m/%d/%y @ %I:%M:%S';
+
+    public static $staffNotify = 86400 * 3;
+    public static $autoClose = 86400 * 5;
+
+    /** @var System $system */
+    protected $system;
+
+    public $user_id;
+    public $support_level;
```
Comment (levimeahan) [inline classes/SupportManager.php:48]:
```suggestion
    public $user_support_role;
```

File: classes/SupportManager.php [hunk 3]
```diff
@@ -0,0 +1,689 @@
...
+        }
+        else {
+            $query = "SELECT * ";
+        }
+
+        $query .= "FROM `support_request`";
+
+        if(!empty($variables)) {
+            $query .= " WHERE ";
+            foreach($variables as $col => $val) {
```
[the remark this answers is not in the document]
Comment (HitoriWoB) [reply classes/SupportManager.php:374]: Everything is currently cleaned before being passed to the search function or is hard coded.
Could clean again as a precaution.

**Review by levimeahan — APPROVED** [review]

**Pull request merged**